optimum/intel/__init__.py

```diff
@@ -121,6 +121,7 @@
             "OVModelForQuestionAnswering",
             "OVModelForSeq2SeqLM",
             "OVModelForSpeechSeq2Seq",
+            "OVModelForVision2Seq",
```

Comment:
can you add `OVModelForVision2Seq` here as well https://github.com/huggingface/optimum-intel/blob/a3bf1727b78056283a4ad0cf77b6a0ff7519aeea/optimum/intel/__init__.py#L242

Reply:
added

Reply:
not updated on my side!

Reply:
please chack one more time